src/coord/src/timestamp.rs

```diff
@@ -814,7 +814,7 @@ impl Timestamper {
                 match get_kafka_partitions(
                     &consumer.consumer,
                     &timestamp_topic,
-                    Duration::from_secs(1),
+                    Duration::from_secs(5),
```

Comment:
was this to make a test less flakey or some other reason?

Reply:
When running things on my machine I would often get the timeout warning. I think it's better to not worry users.